modules/cms/twig/FrameworkNode.php

```diff
@@ -25,13 +25,20 @@ public function __construct($name, $lineno, $tag = 'framework')
     public function compile(TwigCompiler $compiler)
     {
         $attrib = $this->getAttribute('name');
-        $includeExtras = strtolower(trim($attrib)) == 'extras';
 
         $compiler
             ->addDebugInfo($this)
             ->write("\$_minify = ".CombineAssets::class."::instance()->useMinify;" . PHP_EOL);
 
-        if ($includeExtras) {
+        if (strtolower(trim($attrib)) === 'jquery') {
```

Comment:
This is should be handled under the `{% jquery %}` tag, not the framework tag

Reply:
you said to remove it

Reply:
posted that comment before the other one, sorry for the confusion